src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GoToGCPConsoleStep.ts

```diff
@@ -0,0 +1,39 @@
+import { h } from 'react-hyperscript-helpers'
+import { ButtonOutline } from 'src/components/common'
+import { Ajax } from 'src/libs/ajax'
+import Events from 'src/libs/events'
+import * as Utils from 'src/libs/utils'
+import { Step } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/Step'
+import { StepFieldLegend, StepFields } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepFields'
+import { StepHeader } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepHeader'
+
+
+interface GoToGCPConsoleStepProps {
+  isActive: boolean
+  stepFinished: () => void
+}
+
+export const GoToGCPConsoleStep = ({ isActive, ...props }: GoToGCPConsoleStepProps) => {
+  return h(Step, { isActive }, [
+    StepHeader({ title: 'STEP 1' }),
+    h(StepFields, [
+      h(StepFieldLegend, ['Go to the Google Cloud Platform Billing Console and sign-in with the same user you use to login to Terra.']),
```

Comment:
I don't think this should be in a legend/field set, as there are no form options.

Reply:
good point

Reply:
I changed this by introducing `StepFieldInfo`, which might not be the greatest way to do handle this, but easily allowed consistent styling without forcing everything to be in a field set.